tests/appsec/iast/test_iast_sql_injection.py

```diff
@@ -9,8 +9,8 @@
 @released(
     dotnet="?",
     golang="?",
-    java={"spring-boot": "0.108.0", "spring-boot-jetty": "0.108.0", "spring-boot-openliberty": "0.108.0", "*": "?"},
-    nodejs={"express4": "3.6.0", "*": "?"},
+    java="?",
+    nodejs="?",
```

Comment:
@uurien If/when you have SQL injection merged, and its system tests implemented, you'll need to change the version here.

Reply:
SQL injection and system test are implemented and merged, but not released.